src/helpers/Sentry.ts

```diff
@@ -33,7 +36,7 @@ export default {
           new Integrations.CaptureConsole(),
           new Integrations.Dedupe(),
         ],
-        release: `rex@${config.RELEASE_ID}`,
+        release: normalizeReleaseId(`rex@${config.RELEASE_ID}`),
```

Comment:
`dist` is also causing errors.
![image](https://user-images.githubusercontent.com/36712200/110821366-49cceb80-8290-11eb-8227-3cb6f6795d70.png)